content/news/posts/2024/prestashop-news-where-we-are-and-where-we-are-going.md

```diff
@@ -0,0 +1,69 @@
+
+<img src="/assets/images/2024/01/ps-where-we-are-overview-img2.png" style="height:400px; margin: 20px auto;">
+
+Another way to look at this topic is to check not only the PRs merged in one version but also during a year. I actually prefer this chart because the previous one does not acknowledge the contributions made outside of the PrestaShop core repository: the work done on modules, documentation, websites, services, and libraries is omitted. In that sense, this chart is more honest about the full project activity.
+
+This chart shows that we had a high peak of contribution in 2020, and since then, contribution numbers have been steady.
+
+However, on this chart, you’ll find something else I truly appreciate. The chart shows two columns: the first is the total number of contributions, no matter who is the author, and the second one excludes the people from the Open Source Tribe of PrestaShop SA, which is the main task force appointed by the PrestaShop company to work on the software. People in the Open Source Tribe of PrestaShop SA (whom I belong to) are paid full-time to contribute, hence their impact on the number of PRs. So the second column shows PRs authored by people whose job is not targeted at contribution, who contribute because they see it useful. This number has been growing every year since 2019 and has reached an all time record in 2023: more than 1000 Pull Requests merged! This demonstrates that contributions do not have to be money-driven.
+
+### Committers number evolution
+
+<img src="/assets/images/2024/01/ps-where-we-are-overview-img3.png" style="height:400px; margin: 20px auto;">
+
+This final chart shows the evolution of [committers](https://www.prestashop-project.org/project-organization/people-and-roles/#committers) and [members of the technical council](https://www.prestashop-project.org/project-organization/people-and-roles/#technical-council) in the last 2 years. Again I made a distinction between people employed by PrestaShop SA and people who are not. Both numbers of committers paid by PrestaShop SA and other non-employees of the company have grown quite well in 2 years, a testimony of how the project attracts new people every year. 20 people joined the committers team in 2 years, however not all of them are still active in the project.
+
+## PrestaShop 9.0.0 takeaways
+
+### Symfony and PHP
+
+
+PrestaShop 9.0.0 will be a new major version, and this means it will introduce new breaking changes. Among them, one was very important: bumping the version of Symfony used as PrestaShop 8.x relies on Symfony 4, [which is now EOL](https://symfony.com/releases/4.4).
```

Comment:
```suggestion
PrestaShop 9.0.0 will be a new major version, and this means it will introduce new breaking changes. Among them, one was very important: bumping the version of Symfony used, as PrestaShop 8.x relies on Symfony 4, [which is now EOL](https://symfony.com/releases/4.4).
```